docker-compose.yml

```
@@ -12,7 +12,7 @@ services:
       test: echo $$(wget --server-response http://nats:8222/varz 2>&1 | grep  '200 OK') | grep '200'  || exit 1
 
   redis_persistent:
-    image: redis:5-alpine
+    image: redis:7.2.3-alpine
```

Let's only use the major tag version `redis:7-alpine` all across.
```suggestion
    image: redis:7-alpine
```